mmv1/products/securitycenterv2/ProjectNotificationConfig.yaml

```diff
@@ -0,0 +1,127 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ProjectNotificationConfig'
+base_url: projects/{{project}}/locations/{{location}}/notificationConfigs
+self_link: '{{name}}'
+create_url: projects/{{project}}/locations/{{location}}/notificationConfigs?configId={{config_id}}
+update_verb: :PATCH
+update_mask: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/notificationConfigs/{{config_id}}'
+description: |
+  A Cloud Security Command Center (Cloud SCC) notification configs. A
+  notification config is a Cloud SCC resource that contains the
+  configuration to send notifications for create/update events of
+  findings, assets and etc.
+  ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled
+  in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
+  Without doing so, you may run into errors during resource creation.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/projects.locations.notificationConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_project_notification_config_basic'
+    primary_resource_id: 'custom_notification_config'
+    vars:
+      topic_name: 'my-topic'
+      config_id: 'my-config'
+    test_env_vars:
+      project: :PROJECT_NAME
+    ignore_read_extra:
+      - 'location'
+      - 'project'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/self_link_as_name_set_project.go.erb
+  post_create: templates/terraform/post_create/set_computed_name.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: configId
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      This must be unique within the project.
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    immutable: true
+    required: false
+    url_param_only: true
+    default_value: 'global'
+    description: |
+      Location ID of the parent organization. Only global is supported at the moment.
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: |
+      The resource name of this notification config, in the format
+      `projects/{{projectId}}/locations/{{location}}/notificationConfigs/{{config_id}}`.
+  - !ruby/object:Api::Type::String
+    name: description
+    description: |
+      The description of the notification config (max of 1024 characters).
+    validation: !ruby/object:Provider::Terraform::Validation
+      function: 'validation.StringLenBetween(0, 1024)'
+  - !ruby/object:Api::Type::String
+    name: pubsubTopic
+    required: true
```

Comment:
Looking over the docs, it looks like it shouldn't be marked as required. Is their other documentation that supports having it be set to required? docs: https://cloud.google.com/security-command-center/docs/reference/rest/v2/projects.locations.notificationConfigs

Reply:
I don't think there is any other documentation and the current documentation also are vague which doesn't explicitly mention if they are required or not. Based on other contemporary config files I have put it as required, I can remove the field. Let me know if I should remove it.

Reply:
Let's remove it so that it matches the documentation.

Reply:
Removed.